posts/2021-03-18-async-vision-doc.md

```diff
@@ -0,0 +1,72 @@
+
+For each character, we will write a series of ["status quo" stories][sq] that describe the challenges they face as they try to achieve their goals (and typically fail in dramatic fashion!) **These stories are not fiction.** They are an amalgamation of the real experiences of people using Async Rust, as reported to us by interviews, blog posts, and tweets. To give you an idea, we have a draft story where [Grace has to debug a custom future that she wrote][gsq]. We expect to improve this story and add more, both for Grace and other characters.
+
+Writing the "status quo" stories helps us to compensate for the [curse of knowledge][cok]: the folks working on Async Rust tend to be experts in Async Rust. We've gotten used to the [workarounds] required to be productive, and we know the little tips and tricks that can get you out of a jam. The stories help us gauge the cumulative impact all the paper cuts can have on someone still learning their way around. This gives us the data we need to prioritize.
+
+### ...and then tells how we will change it
+
+The ultimate goal of the vision doc, of course, is not just to tell us where we are now, but where we are going and how  we will get there. Once we've made good progress on the status quo stories, the next step will be start brainstorming stories about the ["shiny future"][sf].
+
+The idea is to replay the same scenarios from the "status quo" document, except 2 or 3 years in the future. For example, maybe Grace has access to a debugging tool that is able to diagnose her stuck tasks and tell her what kind of future they are blocked on, so she doesn't have to grep through the logs. Probably she has better ways to do her logging, too, and better documentation to help her get started. Let's be ambitious!
+
+### Involving the whole community
+
+The async vision document provides a forum where the Async Rust community can plan a great overall experience for Async Rust users. Async Rust was intentionally designed not to have a "one size fits all" mindset, and we don't want to change that. Our goal is to build a shared vision for the end-to-end experience while retaining the loosely coupled, exploration-oriented ecosystem we have built.
+
+The [process][htv] we are using to write the vision doc encourages active collaboration and "position sum" thinking. It begins with a [brainstorming period][b] in which we are inviting people to describe their experiences as "status quo" stories; later on, we'll start accepting "shiny future" stories. At the end of the brainstorming period, we will be giving [awards] and then begin work on [harmonizing] the stories into a coherent whole.
```

Comment:
I think the idea of awards is... intriguing, but I am also a bit worried as it feels very low-key mentioned here, while not being a common practice in the Rust project up till now - experimenting is good, but I'm not sure this is quite how to announce/try out a new thing. I would like to see some discussion or notes at least on what scope you're suggesting for the awards, maybe? Right now it just says TBD for the prizes for example, which feels a bit unfortunate.

Reply:
My interpretation of this was that these were more of "kudos" awards than ones where the prizes were the important part, and I kinda think that that's okay to experiment with. Would be good to get some clarification on what the intent is here, and perhaps the blog text can hedge a little bit on this.

Reply:
I like what @Manishearth suggested. I think "kudos" is perfect, basically citing the winners in the closing blog post we will post. I'll try to tweak the language to clarify that.

Reply:
Done.